lib/config/optionsconfiguration.go

```diff
@@ -43,6 +43,7 @@ type OptionsConfiguration struct {
 	TempIndexMinBlocks      int      `xml:"tempIndexMinBlocks" json:"tempIndexMinBlocks" default:"10"`
 	UnackedNotificationIDs  []string `xml:"unackedNotificationID" json:"unackedNotificationIDs"`
 	TrafficClass            int      `xml:"trafficClass" json:"trafficClass"`
+	WeakHashSelectionMethod int      `xml:"weakHashSelectionMethod" json:"weakHashSelectionMethod"`
```

Comment:
Yeah, sadly util.SetDefaults won't be able to handle that, unless some really dark witchcraft.